controllers/apply/lib/field-types/address.js

```diff
@@ -124,12 +118,7 @@ class AddressField extends Field {
 
     get displayValue() {
         if (this.value) {
-            return compact([
-                this.value.line1,
-                this.value.townCity,
-                this.value.county,
-                this.value.postcode
-            ]).join(',\n');
+            return compact(Object.values(this.value)).join(',\n');
```

Comment:
Realised that the fix in https://github.com/biglotteryfund/blf-alpha/pull/2780 needed to apply here too. No need to pick out all the values manually when we can use Object.values.